### PR TITLE
testing(translate): parameterize the timeout

### LINKED ISSUE #4239: translate_v3_batch_translate_text_with_glossary_test: test_batch_translate_text_with_glossary failed
This test failed!

To configure my behavior, see [the Build Cop Bot documentation](https://github.com/googleapis/repo-automation-bots/tree/master/packages/buildcop).

If I'm commenting on this issue too often, add the `buildcop: quiet` label and
I will stop commenting.

---

commit: 51a66f94512b1416dcf0a02de5b787a808419f4d
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/af88574d-f7bc-4b4e-812a-abb561592a34), [Sponge](http://sponge2/af88574d-f7bc-4b4e-812a-abb561592a34)
status: failed
<details><summary>Test output</summary><br><pre>target = functools.partial(<bound method PollingFuture._done_or_raise of <google.api_core.operation.Operation object at 0x7f0d220e70f0>>)
predicate = <function if_exception_type.<locals>.if_exception_type_predicate at 0x7f0d25184400>
sleep_generator = <generator object exponential_sleep_generator at 0x7f0d220c6e08>
deadline = 180, on_error = None

    def retry_target(target, predicate, sleep_generator, deadline, on_error=None):
        """Call a function and retry if it fails.
    
        This is the lowest-level retry helper. Generally, you'll use the
        higher-level retry helper :class:`Retry`.
    
        Args:
            target(Callable): The function to call and retry. This must be a
                nullary function - apply arguments with `functools.partial`.
            predicate (Callable[Exception]): A callable used to determine if an
                exception raised by the target should be considered retryable.
                It should return True to retry or False otherwise.
            sleep_generator (Iterable[float]): An infinite iterator that determines
                how long to sleep between retries.
            deadline (float): How long to keep retrying the target. The last sleep
                period is shortened as necessary, so that the last retry runs at
                ``deadline`` (and not considerably beyond it).
            on_error (Callable[Exception]): A function to call while processing a
                retryable exception.  Any error raised by this function will *not*
                be caught.
    
        Returns:
            Any: the return value of the target function.
    
        Raises:
            google.api_core.RetryError: If the deadline is exceeded while retrying.
            ValueError: If the sleep generator stops yielding values.
            Exception: If the target raises a method that isn't retryable.
        """
        if deadline is not None:
            deadline_datetime = datetime_helpers.utcnow() + datetime.timedelta(
                seconds=deadline
            )
        else:
            deadline_datetime = None
    
        last_exc = None
    
        for sleep in sleep_generator:
            try:
>               return target()

.nox/py-3-6/lib/python3.6/site-packages/google/api_core/retry.py:184: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <google.api_core.operation.Operation object at 0x7f0d220e70f0>

    def _done_or_raise(self):
        """Check if the future is done and raise if it's not."""
        if not self.done():
>           raise _OperationNotComplete()
E           google.api_core.future.polling._OperationNotComplete

.nox/py-3-6/lib/python3.6/site-packages/google/api_core/future/polling.py:84: _OperationNotComplete

The above exception was the direct cause of the following exception:

self = <google.api_core.operation.Operation object at 0x7f0d220e70f0>
timeout = 180

    def _blocking_poll(self, timeout=None):
        """Poll and wait for the Future to be resolved.
    
        Args:
            timeout (int):
                How long (in seconds) to wait for the operation to complete.
                If None, wait indefinitely.
        """
        if self._result_set:
            return
    
        retry_ = self._retry.with_deadline(timeout)
    
        try:
>           retry_(self._done_or_raise)()

.nox/py-3-6/lib/python3.6/site-packages/google/api_core/future/polling.py:104: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

args = (), kwargs = {}
target = functools.partial(<bound method PollingFuture._done_or_raise of <google.api_core.operation.Operation object at 0x7f0d220e70f0>>)
sleep_generator = <generator object exponential_sleep_generator at 0x7f0d220c6e08>

    @general_helpers.wraps(func)
    def retry_wrapped_func(*args, **kwargs):
        """A wrapper that calls target function with retry."""
        target = functools.partial(func, *args, **kwargs)
        sleep_generator = exponential_sleep_generator(
            self._initial, self._maximum, multiplier=self._multiplier
        )
        return retry_target(
            target,
            self._predicate,
            sleep_generator,
            self._deadline,
>           on_error=on_error,
        )

.nox/py-3-6/lib/python3.6/site-packages/google/api_core/retry.py:286: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

target = functools.partial(<bound method PollingFuture._done_or_raise of <google.api_core.operation.Operation object at 0x7f0d220e70f0>>)
predicate = <function if_exception_type.<locals>.if_exception_type_predicate at 0x7f0d25184400>
sleep_generator = <generator object exponential_sleep_generator at 0x7f0d220c6e08>
deadline = 180, on_error = None

    def retry_target(target, predicate, sleep_generator, deadline, on_error=None):
        """Call a function and retry if it fails.
    
        This is the lowest-level retry helper. Generally, you'll use the
        higher-level retry helper :class:`Retry`.
    
        Args:
            target(Callable): The function to call and retry. This must be a
                nullary function - apply arguments with `functools.partial`.
            predicate (Callable[Exception]): A callable used to determine if an
                exception raised by the target should be considered retryable.
                It should return True to retry or False otherwise.
            sleep_generator (Iterable[float]): An infinite iterator that determines
                how long to sleep between retries.
            deadline (float): How long to keep retrying the target. The last sleep
                period is shortened as necessary, so that the last retry runs at
                ``deadline`` (and not considerably beyond it).
            on_error (Callable[Exception]): A function to call while processing a
                retryable exception.  Any error raised by this function will *not*
                be caught.
    
        Returns:
            Any: the return value of the target function.
    
        Raises:
            google.api_core.RetryError: If the deadline is exceeded while retrying.
            ValueError: If the sleep generator stops yielding values.
            Exception: If the target raises a method that isn't retryable.
        """
        if deadline is not None:
            deadline_datetime = datetime_helpers.utcnow() + datetime.timedelta(
                seconds=deadline
            )
        else:
            deadline_datetime = None
    
        last_exc = None
    
        for sleep in sleep_generator:
            try:
                return target()
    
            # pylint: disable=broad-except
            # This function explicitly must deal with broad exceptions.
            except Exception as exc:
                if not predicate(exc):
                    raise
                last_exc = exc
                if on_error is not None:
                    on_error(exc)
    
            now = datetime_helpers.utcnow()
    
            if deadline_datetime is not None:
                if deadline_datetime <= now:
                    six.raise_from(
                        exceptions.RetryError(
                            "Deadline of {:.1f}s exceeded while calling {}".format(
                                deadline, target
                            ),
                            last_exc,
                        ),
>                       last_exc,
                    )

.nox/py-3-6/lib/python3.6/site-packages/google/api_core/retry.py:206: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

value = None, from_value = _OperationNotComplete()

>   ???
E   google.api_core.exceptions.RetryError: Deadline of 180.0s exceeded while calling functools.partial(<bound method PollingFuture._done_or_raise of <google.api_core.operation.Operation object at 0x7f0d220e70f0>>), last exception:

<string>:3: RetryError

During handling of the above exception, another exception occurred:

capsys = <_pytest.capture.CaptureFixture object at 0x7f0d2204d6a0>
bucket = <Bucket: tmp-345cb7a7dfce4c27a5116d4e7c04fd3e>
glossary = 'test-8763692c-e018-4ef7-b87e-b4a4df543853'

    @pytest.mark.flaky(max_runs=3, min_passes=1)
    def test_batch_translate_text_with_glossary(capsys, bucket, glossary):
        translate_v3_batch_translate_text_with_glossary.batch_translate_text_with_glossary(
            "gs://cloud-samples-data/translation/text_with_glossary.txt",
            "gs://{}/translation/BATCH_TRANSLATION_OUTPUT/".format(bucket.name),
            PROJECT_ID,
>           glossary,
        )

translate_v3_batch_translate_text_with_glossary_test.py:75: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
translate_v3_batch_translate_text_with_glossary.py:68: in batch_translate_text_with_glossary
    response = operation.result(180)
.nox/py-3-6/lib/python3.6/site-packages/google/api_core/future/polling.py:125: in result
    self._blocking_poll(timeout=timeout)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <google.api_core.operation.Operation object at 0x7f0d220e70f0>
timeout = 180

    def _blocking_poll(self, timeout=None):
        """Poll and wait for the Future to be resolved.
    
        Args:
            timeout (int):
                How long (in seconds) to wait for the operation to complete.
                If None, wait indefinitely.
        """
        if self._result_set:
            return
    
        retry_ = self._retry.with_deadline(timeout)
    
        try:
            retry_(self._done_or_raise)()
        except exceptions.RetryError:
            raise concurrent.futures.TimeoutError(
>               "Operation did not complete within the designated " "timeout."
            )
E           concurrent.futures._base.TimeoutError: Operation did not complete within the designated timeout.

.nox/py-3-6/lib/python3.6/site-packages/google/api_core/future/polling.py:107: TimeoutError</pre></details>

### PR DESCRIPTION
**Pull request by tmatsuo**
fixes #4239
(by specifying a longer timeout)

### CODE REVIEW

File: translate/cloud-client/translate_v3_create_glossary.py [hunk 1]
```diff
@@ -51,7 +52,7 @@ def create_glossary(
     # to translate the domain-specific terminology.
     operation = client.create_glossary(parent=parent, glossary=glossary)
 
-    result = operation.result(timeout=180)
+    result = operation.result(timeout)
```
Comment (telpirion) [inline translate/cloud-client/translate_v3_create_glossary.py:55]: We'd like to use named arguments as much as possible.

File: translate/cloud-client/translate_v3_delete_glossary.py [hunk 2]
```diff
@@ -17,15 +17,17 @@
...
+        timeout=180,
 ):
     """Delete a specific glossary based on the glossary ID."""
     client = translate.TranslationServiceClient()
 
     parent = client.glossary_path(project_id, "us-central1", glossary_id)
 
     operation = client.delete_glossary(parent)
-    result = operation.result(timeout=180)
+    result = operation.result(timeout)
```
Comment (telpirion) [inline translate/cloud-client/translate_v3_delete_glossary.py:30]: Please convert to named argument.

File: translate/cloud-client/translate_v3_batch_translate_text_with_glossary.py [hunk 3]
```diff
@@ -65,7 +66,7 @@ def batch_translate_text_with_glossary(
     )
 
     print(u"Waiting for operation to complete...")
-    response = operation.result(180)
+    response = operation.result(timeout)
```
Comment (telpirion) [inline translate/cloud-client/translate_v3_batch_translate_text_with_glossary.py:69]: Please use a named argument.

File: translate/cloud-client/translate_v3_batch_translate_text.py [hunk 4]
```diff
@@ -46,7 +47,7 @@ def batch_translate_text(
         output_config=output_config)
 
     print(u"Waiting for operation to complete...")
-    response = operation.result(180)
+    response = operation.result(timeout)
```
Comment (telpirion) [inline translate/cloud-client/translate_v3_batch_translate_text.py:50]: Please use named argument.

**Comment by tmatsuo** [comment]
@telpirion I changed to the named args

**Pull request merged**